web/app.php

```diff
@@ -7,15 +7,17 @@
 
 // Use APC for autoloading to improve performance
 // Change 'sf2' by the prefix you want in order to prevent key conflict with another application
-/*
-$loader = new ApcClassLoader('sf2', $loader);
-$loader->register(true);
-*/
+//$loader = new ApcClassLoader('sf2', $loader);
+//$loader->register(true);
 
 require_once __DIR__.'/../app/AppKernel.php';
 //require_once __DIR__.'/../app/AppCache.php';
+$env = array_key_exists('ORO_ENV', $_SERVER) ? $_SERVER['ORO_ENV'] : 'prod';
```

Comment:
This is development setup it might be valid to set the `environment` and `debug` mode with virtualhost as it is a convenient way to define things. If not set, the production values will be used.